pkg/extensions/search/common/common_test.go

```diff
@@ -28,8 +30,7 @@ var (
 )
 
 const (
-	BaseURL1 = "http://127.0.0.1:8085"
-	Port1    = "8085"
+	BaseURL = "http://127.0.0.1:%s"
```

Comment:
Unless you are referring to this directly, just move this into getBaseURL() itself.

Reply:
Same comment in other test files as well.